web_services/foxx/brightid5/operations.js

```diff
@@ -146,9 +149,15 @@ function verify(op) {
 }
 
 function apply(op) {
-  if (op['name'] == 'Add Connection') {
+  if (op['name'] == 'Connect') {
+    return db.connect(op.id1, op.id2, op.level, op.reportReason, op.replacedWith, op.timestamp);
```

Comment:
I like this to match methods that are called by other operations.
for methods that are called on other simpler operations, it's cleaner to have different parameters instead of passing op.

Reply:
>it's cleaner to have different parameters instead of passing op.
I think it's cleaner to pass `op` and also future-proof. You wouldn't even have to make this change now if you had been doing that, but you are writing the code so you get to choose. Maybe if make some changes to the code in the future I will change it ;).